test/unit/discovered_extensions_test.rb

```diff
@@ -70,11 +70,42 @@ class FindDiscoveryRulesTest < ActiveSupport::TestCase
     host = Host::Discovered.import_host_and_facts(facts).first
     FactoryGirl.create(:discovery_rule, :priority => 1, :search => "facts.somefact = x",
                        :organizations => [host.organization], :locations => [host.location])
-    r2 = FactoryGirl.create(:discovery_rule, :priority => 2, :search => "facts.somefact = abc",
+    r2 = FactoryGirl.create(:discovery_rule, :priority => 1, :search => "facts.somefact = abc",
                             :organizations => [host.organization], :locations => [host.location])
     assert_equal find_discovery_rule(host), r2
```

Comment:
I need to test cases with same priorities. Now we have two tests covering both 1st and 2nd rule. I reused existing test because it was testing wrong case - fact did not match therefore it was a simple case that is already covered above.